.github/workflows/auto-comment.yml

```diff
@@ -16,25 +16,32 @@ jobs:
             in its title, or leave it as a github draft.
 
 
-            Please make sure it is compliant with our [contributing guidelines](https://nilearn.github.io/stable/development.html#contribution-guidelines).
+            Please make sure it is compliant 
+            with our [contributing guidelines](https://nilearn.github.io/stable/development.html#contribution-guidelines).
             In particular, be sure it checks the boxes listed below.
 
             - [ ] PR has an interpretable title.
 
-            - [ ] PR links to Github issue with mention "Closes #XXXX"
+            - [ ] PR links to Github issue with mention `Closes #XXXX`
+                  (see our documentation on [PR structure](https://nilearn.github.io/stable/development.html#pr-structure))
 
-            - [ ] Code is PEP8-compliant.
+            - [ ] Code is PEP8-compliant
+                  (see our documentation on [coding style](https://nilearn.github.io/stable/development.html#coding-style))
 
-            - [ ] Changelog or what's new entry in "doc/changes/latest.rst"
+            - [ ] Changelog or what's new entry in `doc/changes/latest.rst`
+                  (see our documentation on [changelogs](https://nilearn.github.io/stable/development.html#changelog))
```

Comment:
This won't work for a while because it's linking the stable docs (so until the next release in maybe 6 months). Maybe for now just link the pr-structure it's under and we can change it later

Reply:
Done.

Also added a TODO as reminder.

Reply:
Yes you can link it to the dev doc instead so: https://nilearn.github.io/dev/development.html#changelog

I guess in this case that's ok since contributors might already be familiar with the development version

Reply:
> Done.
> 
> Also added a TODO as reminder.

Ok that's fine